src/styles/index.css

```diff
@@ -0,0 +1,130 @@
+body {
+  margin: 0;
+  font-family: Roboto, sans-serif;
+}
+
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  background-image: url(../images/star.svg);
+  width: 16px;
+  height: 16px;
+  background-repeat: no-repeat;
+  background-position: center;
+  margin-left: 4px;
+}
+
+.stars__star:first-child {
+  margin-left: 0;
+}
+
+.stars--1 .stars__star:nth-child(-n + 1),
+.stars--2 .stars__star:nth-child(-n + 2),
+.stars--3 .stars__star:nth-child(-n + 3),
+.stars--4 .stars__star:nth-child(-n + 4),
+.stars--5 .stars__star:nth-child(-n + 5) {
+  background-image: url(../images/star-active.svg);
+}
+
+.card {
+  display: flex;
+  width: 200px;
+  border: 1px solid #f3f3f3;
+  box-sizing: border-box;
+  justify-content: center;
+  flex-direction: column;
+}
+
+.card_info {
+  text-align: left;
+  padding: 0 16px;
+}
+
+.card_imac--img {
+  margin-top: 32px;
+  margin-bottom: 40px;
+  width: 160px;
+  height: 134px;
+  padding: 0 19px;
```

Comment:
According to the code style rules, 'padding' should not be used with 'width' or 'height'. Consider adjusting the element's box model or the surrounding context to achieve the desired layout.